src/components/message-input/giphy/styles.scss

```diff
@@ -0,0 +1,54 @@
+@use '../../../shared-components/theme-engine/theme' as themeDeprecated;
+
+.giphy__container {
+  width: 500px;
+  position: absolute;
+  bottom: 65px;
+  right: 74px;
+  background-color: themeDeprecated.$background-color-tertiary;
+  padding: 5px;
+  border-radius: 5px;
+  border: 1px solid themeDeprecated.$input-arrow-color;
+
+  &--fullscreen {
+    right: 65px;
+  }
```

Comment:
It feels very strange that a component at this low level needs to know whether it's fullscreen or not. Feels like we may have previously defined the positioning poorly and this is just a bandaid. Positioning

Let's leave it for now but keep this in mind for the future. Full screen mode should impact the major layout pieces but not lower level pieces directly. I think there's a missing concept here or just a funky lower level layout thing that we're not using properly.